src/consensus/malachite/spawn.rs

```diff
@@ -181,9 +182,12 @@ impl MalachiteConsensusActors {
         )
         .await?;
 
+        let mut timeout_config = TimeoutConfig::default();
+        // setting the commit timeout sets up a fixed block production rate.
```

Comment:
This affects and slows down sync right now, but malachite has already fixed this. So we should get it when we upgrade next week.